.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,62 @@
+<!--
+Before reporting a new issue, please make sure that it's not a duplicate.
+
+If you suspect your issue is a bug, please provide information as shown below. If your issue is a feature request, this information is not always necessary.
+-->
+
+**Description**
+
+<!--
+Briefly describe the problem you are having in a few paragraphs.
+-->
+
+**Steps to reproduce the issue:**
+1.
+2.
+3.
+
+**Describe the results you received:**
+
+
+**Describe the results you expected:**
+
+
+**Additional information you deem important (e.g. issue happens only occasionally):**
+
+**CRIU logs and information:**
+
+<!--
+You can either attach logs as files to the issue or put them under details
+-->
+
+<details><summary>CRIU full dump/restore logs:</summary>
+<p>
+
+```
+(paste your output here)
+```
```

Comment:
The logs could be uploaded as a txt file. This is easier because the `-o` option can be used to create such file(s).

Reply:
I've already added a comment in the begining of "CRIU logs and information" section that logs can be either downloaded or put under <details>. I don't want to over-restrict people, for some it would be easier just to paste the log...